### PR TITLE
Cloud pc/add windows settings

### PR DESCRIPTION
**Pull request by Guoan-Tang**
add windowsSettings into CloudPCOrganizationSetting

### CODE REVIEW

**Review by RuiHou105 — CHANGES_REQUESTED** [review]
Hi @Guoan-Tang , could you please create a roadmap workitem and attach it in the description? You can follow this [guidance ](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211772/Content-review-process-summary).  Here is a roadmap workitem example https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/11551/

**Comment by Guoan-Tang** [comment]
**Roadmap workitem**:

https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/13582
[Devices and apps][Cloud PC] Add properties windowsSettings to cloudPcOrganizationSettings

**Review by AshleyYangSZ — APPROVED** [review]

**Review by RuiHou105 — APPROVED** [review]

**Review by Danielabom — APPROVED** [review]
This PR has been reviewed.

File: api-reference/beta/api/cloudpcorganizationsettings-update.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ PATCH /deviceManagement/virtualEndpoint/organizationSettings
 |:---|:---|:---|
 |osVersion|cloudPcOperatingSystem|The version of the operating system (OS) to provision on Cloud PCs. The possible values are: `windows10`, `windows11`, `unknownFutureValue`. Optional.|
 |userAccountType|cloudPcUserAccountType|The account type of the user on provisioned Cloud PCs. The possible values are: `standardUser`, `administrator`, `unknownFutureValue`. Optional.|
+|windowsSettings|cloudPcWindowsSettings|The settings to apply to Windows while creating Cloud PCs for this organization. The default value is a WindowsSettings with default language value "en-US".|
```
Comment (Lauragra) [inline api-reference/beta/api/cloudpcorganizationsettings-update.md:50]: The second sentence is a little confusing - by this do we mean, "The default language value is `en-us`? If so, please update.

File: api-reference/beta/resources/cloudpcorganizationsettings.md [hunk 2]
```diff
@@ -27,6 +27,7 @@ Represents the Cloud PC organization settings for a tenant. A tenant has only on
 |id|String|The ID of the organization settings.|
 |osVersion|[cloudPcOperatingSystem](#cloudpcoperatingsystem-values)|The version of the operating system (OS) to provision on Cloud PCs. The possible values are: `windows10`, `windows11`, `unknownFutureValue`.|
 |userAccountType|[cloudPcUserAccountType](#cloudpcuseraccounttype-values)|The account type of the user on provisioned Cloud PCs. The possible values are: `standardUser`, `administrator`, `unknownFutureValue`.|
+|windowsSettings|[cloudPcWindowsSettings](../resources/cloudpcwindowssettings.md)|Represents the Cloud PC organization settings for a tenant. A tenant has only one cloudPcOrganizationSettings object. The default language value `en-US`.|
```
Comment (Lauragra) [inline api-reference/beta/resources/cloudpcorganizationsettings.md:30]: Please make the resource name cloudPcOrganizationSettings bold.

File: api-reference/beta/api/cloudpcorganizationsettings-update.md (hunk 1 above)
Comment (Guoan-Tang) [reply api-reference/beta/api/cloudpcorganizationsettings-update.md]: Nice point, had fixed it

**Pull request merged**